sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -351,15 +351,15 @@ public async Task CreateAndGetAlertConfigurationWithMultipleMetricConfigurations
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21177")]
         public async Task UpdateAlertConfigurationWithMinimumSetupAndGetInstance(bool useTokenCrendential)
         {
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient(useTokenCrendential);
 
             // Configure the Metric Anomaly Alert Configurations to be used.
 
             string hookName = Recording.GenerateAlphaNumericId("hook");
-            var hookToCreate = new WebNotificationHook() { Name = hookName, Endpoint = new Uri("http://contoso.com/") };
+            var hookToCreate = new EmailNotificationHook() { Name = hookName, EmailsToAlert = { "fake@email.com" } };
```

Comment:
For these tests, it doesn't matter if we have a web hook or an email hook, as long as we have a hook ID. I'm updating it because our web hook tests are now failing after a recent service change. We were using mock endpoints (`http://contoso.com/`) for testing, but how they added extra validation before accepting it, so we need to set up a test resource for that later (tracked by https://github.com/Azure/azure-sdk-for-net/issues/21504).

Reply:
:(

Reply:
:(